wp-content/plugins/wporg-learn/inc/sensei.php

```diff
@@ -163,3 +163,60 @@ function wporg_fix_learning_mode_header_space() {
 	wp_add_inline_style( 'learning-mode-header-fix', $custom_styles );
 }
 add_action( 'sensei_course_learning_mode_load_theme', __NAMESPACE__ . '\wporg_fix_learning_mode_header_space' );
+
+/**
+ * Add script to count unique learners
+ */
+function wporg_learn_add_student_count_to_reports( $type ) {
+	if ( 'users' !== $type ) {
+		return; // Only show the count on the students report screen.
+	}
+
+	$from_date = \DateTime::createFromFormat( 'Y-m-d', $_GET['from_date'] ?? '', new \DateTimeZone( 'UTC' ) );
+	$to_date   = \DateTime::createFromFormat( 'Y-m-d', $_GET['to_date'] ?? '', new \DateTimeZone( 'UTC' ) );
+
+	global $wpdb;
+	$query = "SELECT COUNT(DISTINCT user_id)
+FROM $wpdb->comments
```

Comment:
@bsanevans could I ask you to indent the second and third lines of the SQL query under the first line, just to make it cleaner.

Reply:
```suggestion
	FROM $wpdb->comments
```